src/main/resources/l10n/Menu_de.properties

```diff
@@ -87,7 +87,7 @@ Import_into_current_database=Importieren_in_aktuelle_&Datenbank
 
 Switch_to_%0_mode=In_den_%0-Modus_wechseln
 Push_entries_to_external_application_(%0)=&Eintr\u00e4ge_in_externe_Anwendung_einf\u00fcgen_(%0)
-Pull_changes_from_shared_database=Entfernte_Änderungen_beziehen
+Pull_changes_from_shared_database=\u00c4nderungen_der_geteilten_Datenbank_beziehen
```

Comment:
Yes,! The tests will fail if you use the encoded \uXXX things.  Make sure to replac them all (I had to change it in the Italian lang files)
e